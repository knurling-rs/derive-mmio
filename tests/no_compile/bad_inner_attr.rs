#[derive(derive_mmio::Mmio)]
#[repr(C)]
struct Uart {
    #[mmio(RW)]
    status: u32,
}

fn main() {
}
