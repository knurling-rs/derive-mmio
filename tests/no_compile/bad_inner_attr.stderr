error: `#[mmio(...)]` only supports 'Inner', 'Read', 'PureRead', 'Write', and 'Modify' options
 --> tests/no_compile/bad_inner_attr.rs:4:5
  |
4 |     #[mmio(RW)]
  |     ^
